src/views/datasets/buoy-telemetry/summary.vue

```diff
@@ -182,8 +202,46 @@ import ExternalLink from "../../../components/base/ExternalLink.vue";
 
 const store = inject("store") as BuoyStore;
 
+const generateQuery = (query) => {
+  const queryParams = Object.entries(query)
+    .map(([key, val]) => `${key}=${val}`)
+    .join("&");
+  return `/dataset/${store.name}/dashboard?${queryParams}`;
+};
+
 const endDate = store.maxDateRaw;
 const startDate = new Date(endDate.valueOf() - 5 * 24 * 60 * 60 * 1000);
+const monthDate = new Date(endDate.valueOf() - 30 * 24 * 60 * 60 * 1000);
```

Comment:
Is this last month as in the last 30 days or the previous calendar month?

If it's the last 30 days, the text should reflect that instead. Maybe something like "Phosphate and Nitrate changes in the last 30 days"

Reply:
Good suggestions, thanks!